include/swift/AST/Decl.h

```diff
@@ -5668,6 +5668,13 @@ class FuncDecl : public AbstractFunctionDecl {
   /// Perform basic checking to determine whether the @IBAction attribute can
   /// be applied to this function.
   bool isPotentialIBActionTarget() const;
+
+  /// SWIFT_ENABLE_TENSORFLOW
+  /// True if the function has `@TensorFlowGraph` attribute, meaning that this
+  /// function corresponds to a TensorFlow graph function.
+  bool isTensorFlowGraphFunction() const {
+    return getAttrs().hasAttribute<TensorFlowGraphAttr>();
```

Comment:
I'd prefer if this would look at the ExtInfo of the function type of the declaration.

Reply:
I used this helper earlier but it's no longer used. Removed.